packages/flutter/lib/src/rendering/view.dart

```diff
@@ -293,9 +298,11 @@ class RenderView extends RenderObject with RenderObjectWithChildMixin<RenderBox>
     final SystemUiOverlayStyle? upperOverlayStyle = layer!.find<SystemUiOverlayStyle>(top);
     // Only android has a customizable system navigation bar.
     SystemUiOverlayStyle? lowerOverlayStyle;
+    bool isAndroid = false;
```

Comment:
This becomes convenient later in the code. Instead of assigning it in the switch statement, I suggest that we assign it with 
```dart
bool isAndroid = defaultTargetPlatform == TargetPlatform.android;
```